backend_new/package.json

```diff
@@ -14,22 +14,29 @@
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "test": "jest",
+    "test": "jest --config ./test/jest.json",
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json",
-    "db:setup": "psql -c 'DROP DATABASE IF EXISTS bloom_prisma;' && psql -c 'CREATE DATABASE bloom_prisma;' && psql -d bloom_prisma -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";' && yarn prisma migrate deploy"
+    "db:resetup": "psql -c 'DROP DATABASE IF EXISTS bloom_prisma;' && psql -c 'CREATE DATABASE bloom_prisma;' && psql -d bloom_prisma -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
+    "db:migration:run": "yarn prisma migrate deploy",
+    "db:seed": "yarn prisma db seed",
+    "generate:client": "ts-node scripts/generate-axios-client.ts && prettier -w types/src/backend-swagger.ts",
+    "test:e2e": "yarn db:resetup && yarn db:migration:run && jest --config ./test/jest-e2e.json",
+    "db:setup": "yarn db:resetup && yarn db:migration:run && yarn db:seed"
```

Comment:
I'm assuming this command will just be used for local development? equivalent to the current "reseed" command?

Reply:
yeah! it does similar things:
destroys the current db, recreates the db, runs migrations, and then seeds data all in one go!